README.md

```diff
@@ -1,368 +1,276 @@
 ```
 
-Quantize floating-point attributes and oct-encode normals
-```
-node ./bin/3d-tiles-tools.js optimizeB3dm -i ./specs/data/batchedWithBatchTableBinary.b3dm -o ./output/optimized.b3dm --options -q -n
-```
+#### cmptToGlb
 
-| Flag | Description | Required |
-| ---- | ----------- | -------- |
-|`-i`, `--input`| Input path of the `.b3dm`| :white_check_mark: Yes |
-|`-o`, `--output`| Output path of the resulting `.b3dm` | No |
-|`-f`, `--force`|Overwrite output file if it exists.| No, default `false` |
-|`--options`|All arguments past this flag are consumed by gltf-pipeline.| No |
+Extracts the glb models from a cmpt tile. If multiple models are found a number will be appended to the output file name.
 
-To use tileset texture compression, pass the [`texcomp` flags](https://github.com/CesiumGS/gltf-pipeline/blob/main/README.md#command-line-flags)
 ```
-node ./bin/3d-tiles-tools.js optimizeB3dm -i ./specs/data/Textured/batchedTextured.b3dm -o ./output/optimized.b3dm --options --texcomp.dxt1.enable --texcomp.dxt1.quality=5 --texcomp.etc1.enable
+npx 3d-tiles-tools b3dmToGlb -i ./specs/data/composite.cmpt -o ./output/extracted.glb
 ```
-This example optimizes the b3dm and compresses the textures into `dxt1` and `etc1` formats.
 
-### optimizeI3dm
+#### optimizeB3dm
 
-Optimize a i3dm using [gltf-pipeline](https://github.com/CesiumGS/gltf-pipeline/blob/main/README.md).
-Since this tool does not process an entire tileset, it cannot be used with the Pipeline tool.
+Optimize a b3dm using [gltf-pipeline](https://github.com/CesiumGS/gltf-pipeline/blob/main/README.md). 
 
 ```
-node ./bin/3d-tiles-tools.js optimizeI3dm -i ./specs/data/instancedWithBatchTableBinary.i3dm -o ./output/optimized.i3dm
+npx 3d-tiles-tools optimizeB3dm -i ./specs/data/batchedWithBatchTableBinary.b3dm -o ./output/optimized.b3dm
 ```
 
-Quantize floating-point attributes and oct-encode normals
-```
-node ./bin/3d-tiles-tools.js optimizeI3dm -i ./specs/data/instancedWithBatchTableBinary.i3dm -o ./output/optimized.i3dm --options -q -n
-```
+Additional command line options:
 
 | Flag | Description | Required |
 | ---- | ----------- | -------- |
-|`-i`, `--input`| Input path of the `.i3dm`| :white_check_mark: Yes |
-|`-o`, `--output`| Output path of the resulting `.i3dm` | No |
-|`-f`, `--force`|Overwrite output file if it exists.| No, default `false` |
 |`--options`|All arguments past this flag are consumed by gltf-pipeline.| No |
 
-To use tileset texture compression, pass the [`texcomp` flags](https://github.com/CesiumGS/gltf-pipeline/blob/main/README.md#command-line-flags).
+**Examples**: 
+
+Quantize floating-point attributes and oct-encode normals:
 ```
-node ./bin/3d-tiles-tools.js optimizeI3dm -i ./specs/data/Textured/instancedTextured.i3dm -o ./output/optimized.i3dm --options --texcomp.dxt1.enable --texcomp.dxt1.quality=5 --texcomp.etc1.enable
+npx 3d-tiles-tools optimizeB3dm -i ./specs/data/batchedWithBatchTableBinary.b3dm -o ./output/optimized.b3dm --options -q -n
```

Comment:
These quantization options are no longer available in recent versions of gltf-pipeline. This example should probably be removed.